tasks/adventure_solutions/task_4_loops_lists.py

```diff
@@ -0,0 +1,58 @@
+import random #import library random
+
+# create list with names
+friends = ['Emily', 'Aurora', 'Jack', 'Daniel', 'Meghan']
+
+def wrong_input(valid):
+    print(f"Sorry but the only valid options are: {valid}.")
+
+print()
+name = input("What is your name? ")
+print(f"Welcome to the dungeon, {name}!")
+
+print("Do you go through door 1 or door 2?")
+door = input("> ")
+
+if door == "1":
+    # print name randomly chosen from list
+    print(f"Look who's here! Your friend {friends[random.randint(0,4)]}!")
+
+    print("There is a nice vampire asking you if you enjoy life.")
+    print("What do you do?")
+    print("1. Smile and nod")
+    print("2. Scream and run")
+
+    vampire = input("> ")
+
+    if vampire == "1":
+        print(f"Congratulations {name}, you found a new friend!")
+    elif vampire == "2":
+        print(f"Sorry {name}, the vampire is faster. You become a dinner.")
+    else:
+        wrong_input("1, 2")
+
+elif door == "2":
+    # print name randomly chosen from list
+    print(f"Look who's here! Your friend {friends[random.randint(0,4)]}!")
```

Comment:
```suggestion
    rand_int = random.randint(0,4)  # this can be also saved in a variable first to make it more readable
    print(f"Look who's here! Your friend {friends[rand_int]}!")
```